smartsim/_core/_install/builder.py

```diff
@@ -342,15 +342,19 @@ def build_from_git(
         bin_path = Path(dependency_path, "bin").resolve()
         try:
             database_exe = next(bin_path.glob("*-server"))
-            database = Path(os.environ.get("REDIS_PATH", database_exe)).resolve()
+            database = Path(
+                os.environ.get("SMARTSIM_REDIS_PATH", database_exe)
+            ).resolve()
             _ = expand_exe_path(str(database))
         except (TypeError, FileNotFoundError) as e:
             raise BuildError("Installation of redis-server failed!") from e
 
         # validate install -- redis-cli
         try:
             redis_cli_exe = next(bin_path.glob("*-cli"))
-            redis_cli = Path(os.environ.get("REDIS_CLI_PATH", redis_cli_exe)).resolve()
+            redis_cli = Path(
+                os.environ.get("SMARTSIM_REDIS_CLI_PATH", redis_cli_exe)
```

Comment:
Since this points to the executable itself, what do you think about renaming to `SMARTSIM_REDIS_CLI_EXE`

Reply:
done!